src/main/java/com/petros/bibernate/dao/EntityPersister.java

```diff
@@ -255,13 +236,16 @@ private static String getUpdatePlaceholders(List<?> updatableColumns) {
                 .collect(Collectors.joining(", "));
     }
 
-    private static <T> T mapResultSetToEntity(Class<T> entityClass, ResultSet resultSet) {
+    private <T> T mapResultSetToEntity(Class<T> entityClass, ResultSet resultSet) {
         try {
+            var fieldValues = new ArrayList<>();
```

Comment:
Looks like fieldValues is not used and could be removed

Reply:
My bad, fixed